resources/assets/less/bem/status-recent.less

```diff
@@ -0,0 +1,43 @@
+/**
+ *    Copyright 2016 ppy Pty. Ltd.
+ *
+ *    This file is part of osu!web. osu!web is distributed with the hope of
+ *    attracting more community contributions to the core ecosystem of osu!.
+ *
+ *    osu!web is free software: you can redistribute it and/or modify
+ *    it under the terms of the Affero GNU General Public License version 3
+ *    as published by the Free Software Foundation.
+ *
+ *    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+ *    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *    See the GNU Affero General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Affero General Public License
+ *    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+@import '../app';
+
+.status-recent {
+  &__uptime, &__incidents {
+    padding: 20px 20px;
+
+    &-title {
+      margin-bottom: 10px;
+      font-size: 18px;
+    }
+
+    h5&--active {

```

Comment:
this should be a modifier of the element below (I couldn't even tell they're actually same element when written this way).